### DISCUSSION TITLE
3Cols chrome app

### DISCUSSION THREAD

**Discussion by sherissar** [OliBlade/3Cols#26] [Q&A] [answered]
Hi, great app, thank you. Where can I find the App Install as a Chrome app on Android and Windows?

**Comment by OliBlade** [comment, marked as answer]
Hi! 3Cols is a PWA (progressive web app) which means when you visit the page on Chrome for Android or Windows you should have a prompt to install as an app.

On Android this usually looks something like this
![Screenshot_20210805-180218](https://user-images.githubusercontent.com/25457955/128392030-4e616f04-0d12-446c-b521-05d4805eaa37.jpg) or sometimes is hidden in the menu on the top right.

On Chrome for Windows the option is usually in the top right of the URL bar.

**Comment by sherissar** [comment]
![Screenshot 2021-08-06 080411](https://user-images.githubusercontent.com/10666064/128463814-5b61ebdb-39c3-48c3-9a51-20bce8762ba5.jpg)
I found it. Thanks!
